lib/isValidationErrorLike.test.ts

```diff
@@ -0,0 +1,29 @@
+import { isValidationErrorLike } from './isValidationErrorLike';
+import { ValidationError } from './ValidationError';
+
+describe('isValidationErrorLike()', () => {
+  test('returns true when argument is an actual instance of ValidationError', () => {
+    expect(isValidationErrorLike(new ValidationError('foobar'))).toEqual(true);
+  });
+
+  test('returns true when argument resembles a ValidationError', () => {
+    const err = new Error('foobar');
+    err.name = 'ZodValidationError'; // force ZodValidationError
```

Comment:
[suggestion] Use this example in the docs under the relevant section

https://github.com/causaly/zod-validation-error/blob/e71198ea9e343c9d91d99700c21f9543a2056504/README.md?plain=1#L186-L187

Reply:
Hm, but this is not what I am trying to do with the test.

Reply:
Maybe my point wasn't clear. I mean to take lines `L10-L11` from the test file, and use them as an example in our docs. Because our docs the way they are, they do not really indicate what's the difference between `isValidationError` and `isValidationErrorLike`.

Reply:
Ah got it. It's debatable though whether our user will be confused by that or not. I mean we have (a) people asking with `fromZodError` doesn't work with any error and (b) people asking why `console.log(err)` prints the entire error and not just the message. I'd rather keep it simple.